context/resolve.go

```diff
@@ -38,7 +38,11 @@ func (ctx *Context) loadPackage() error {
 	ctx.dirty = false
 	ctx.statusCache = nil
 	ctx.Package = make(map[string]*Package, len(ctx.Package))
-	err := filepath.Walk(ctx.RootDir, func(path string, info os.FileInfo, err error) error {
+	rootdir, err := filepath.EvalSymlinks(ctx.RootDir)
```

Comment:
Please add a comment above this line. Explain that this satisfies a use case where the root of the repo is symlinked into the GOPATH, esp for CI cases.